hw/mcu/microchip/pic32mz/src/system_pic32.c

```diff
@@ -42,6 +42,27 @@ static inline int PLL_ODIV(int n)
 
 uint32_t SystemCoreClock;
 
+#if __PIC32_HAS_L1CACHE
+void
+dcache_flush_area(void *addr, int size)
+{
+    uint32_t a = (uint32_t)(addr) & ~3;
+    for (; size > 0; --size, a += 4) {
+        _cache(17, (void *)a);
+    }
+}
+
+void
+dcache_flush(void)
+{
+    for (int i = 0; i < 64; ++i) {
```

Comment:
nitpicking : variables to be declared at beginning of function

Reply:
fixed